Refresh.Interfaces.Game/Endpoints/Levels/PublishEndpoints.cs

```diff
@@ -65,6 +64,14 @@ private static bool VerifyLevel(GameLevelRequest body, DataContext dataContext)
             return false;
         }
 
+        if (!string.IsNullOrWhiteSpace(body.PublisherLabels))
+        {
+            // Unknown labels will be ignored and be missing from the level. No notifications needed because the publisher should
+            // immediately be able to tell if there are missing labels on the level they just published.
+            // Also make sure there are no duplicate labels submitted.
+            body.FinalPublisherLabels = LabelExtensions.FromLbpCommaList(body.PublisherLabels).Distinct().Take(UgcLimits.MaximumLabels);
```

Comment:
Try to avoid chaining LINQ on a single line like this, put each call on a separate line.